.travis.yml

```diff
@@ -34,26 +29,6 @@ script:
 
 matrix:
   exclude:
-    - python: 3.8
-      env: DJANGO="Django>=1.11,<1.12"
-    - python: 2.7
-      env: DJANGO="Django>=2.0,<2.1"
-    - python: 3.8
-      env: DJANGO="Django>=2.0,<2.1"
-    - python: 2.7
-      env: DJANGO="Django>=2.1,<2.2"
-    - python: 3.4
-      env: DJANGO="Django>=2.1,<2.2"
-    - python: 3.8
-      env: DJANGO="Django>=2.1,<2.2"
-    - python: 2.7
-      env: DJANGO="Django>=2.2,<2.3"
-    - python: 3.4
-      env: DJANGO="Django>=2.2,<2.3"
-    - python: 2.7
-      env: DJANGO="Django>=3.0,<3.1"
-    - python: 3.4
-      env: DJANGO="Django>=3.0,<3.1"
     - python: 3.5
       env: DJANGO="Django>=3.0,<3.1"
     - python: 2.7
```

Comment:
Think we can remove all of the exclusions for versions not present in the `python` list above?

Reply:
Yes. Let me have another look.